Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md

```diff
@@ -35,6 +35,87 @@ example, if the Rook operator is running in the namespace `my-namespace` the
 provisioner value should be `my-namespace.rbd.csi.ceph.com`. The same provisioner
 name must be set in both the storageclass and snapshotclass.
 
+search for `# csi-provisioner-name` in the [csi
```

Comment:
The docs actually cannot use a relative path to the code because the docs are generated to a different site. I'd suggest no need for a link, we can have a simple comment like this:
```suggestion
To find the provisioner name in the example storage classes, search for: `# csi-provisioner-name`
```